Adding Policy Evaluations as an exception to "hard-mandatory" (#505)
hashicorp/web-unified-docs: content/terraform-docs-common/docs/cloud-docs/policy-enforcement/manage-policy-sets/index.mdx

- **advisory:** Failed policies never interrupt the run. They provide information about policy check failures in the UI.
- **soft mandatory:** Failed policies stop the run, but any user with [Manage Policy Overrides permission](/terraform/cloud-docs/users-teams-organizations/permissions#manage-policy-overrides) can override these failures and allow the run to complete.

> - **hard mandatory:** Failed policies stop the run. Terraform does not apply runs with failed **hard mandatory** policies until a user fixes the issue that caused the failure.  Some exceptions apply, see documentation on [Policy Evaluations](/terraform/cloud-docs/policy-enforcement/manage-policy-sets#policy-evaluations).

Reviewer: ```suggestion
- **hard mandatory:** Failed policies stop the run. Terraform does not apply runs with failed **hard mandatory** policies until a user fixes the issue that caused the failure. Some exceptions apply. Refer to [Policy Evaluations](/terraform/cloud-docs/policy-enforcement/manage-policy-sets#policy-evaluations) for more information.
```
